gensim/models/keyedvectors.py

```diff
@@ -1383,6 +1383,58 @@ def relative_cosine_similarity(self, wa, wb, topn=10):
 
         return rcs
 
+    def get_keras_embedding(self, train_embeddings=False, word_index=None):
+        """Get a Keras 'Embedding' layer with weights set as the Word2Vec model's learned word embeddings.
+
+        Parameters
+        ----------
+        train_embeddings : bool
+            If False, the weights are frozen and stopped from being updated.
+            If True, the weights can/will be further trained/updated.
+
+        word_index : {str : int}
+            A mapping from tokens to their indices the way they will be provided in the input to the embedding layer.
+            The embedding of each token will be placed in the corresponding index in the embedding matrix.
+            Tokens not in the index are ignored and not placed in the returned layer embedding matrix.
+            This is useful when the token indices are produced by a process that is not coupled with the embedding
+            model, e.x. an Keras Tokenizer object.
+            If None, the embedding matrix in the embedding layer will be indexed according to self.vocab
+
+        Returns
+        -------
+        `keras.layers.Embedding`
+            Embedding layer.
+
+        Raises
+        ------
+        ImportError
+            If `Keras <https://pypi.org/project/Keras/>`_ not installed.
+
+        Warnings
+        --------
+        Current method work only if `Keras <https://pypi.org/project/Keras/>`_ installed.
+
+        """
+        try:
+            from keras.layers import Embedding
+        except ImportError:
+            raise ImportError("Please install Keras to use this function")
+        if word_index is None:
+            weights = self.vectors
+        else:
+            max_index = max(word_index.values())
+            weights = np.random.normal(size=(max_index + 1, self.vectors.shape[1]))
+            for word, index in word_index.items():
+                if word in self.vocab:
+                    weights[index] = self.get_vector(word)
+
+        # set `trainable` as `False` to use the pretrained word embedding
```

Comment:
Who is this comment for?

Reply:
This was just copied from the original code, I just moved it to a new file.